app/styles/app.scss

```diff
@@ -56,7 +56,7 @@
 
 // WU COMPONENTS (DEPRECATED)
 @import 'ember-vo-webuniversum';
-@import 'ember-vo-webuniversum-data-table';
+// @import 'ember-vo-webuniversum-data-table'; // This imports the ember-data-table styles which make <duet-date-picker> not render correctly
```

Comment:
This conflicted with the styles of the date picker. I think it can just be removed though. Everything seems to be converted to AuDataTable already?

Reply:
@Dietr can you confirm we no longer need these styles? (I didn't see anything of so I assume they are no longer used)

Reply:
There was a small issue with the meeting tables but added a fix for that